cmd/logs.go

```diff
@@ -26,106 +28,149 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	maxEsLogsPerQuery     = 10000
+	fourteenDaysInSeconds = 60 * 60 * 24 * 14
+)
+
 // numberOfLines of logfile to return
 var numberOfLines = "400"
 
-// logsCmd represents the logs command
-var logsCmd = &cobra.Command{
-	Use:   "logs (gardener-apiserver|gardener-controller-manager|ui|api|scheduler|controller-manager|etcd-operator|etcd-main[etcd backup-restore]|etcd-main-backup|etcd-events[etcd backup-restore]|addon-manager|vpn-seed|vpn-shoot|machine-controller-manager|dashboard|prometheus|grafana|alertmanager|tf (infra|dns|ingress)",
-	Short: "Show and optionally follow logs of given component\n",
-	Long:  `s`,
-	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) < 1 || len(args) > 2 {
-			fmt.Println("Command must be in the format: logs (gardener-apiserver|gardener-controller-manager|ui|api|scheduler|controller-manager|etcd-operator|etcd-main[etcd backup-restore]|etcd-events[etcd backup-restore]|addon-manager|vpn-seed|vpn-shoot|machine-controller-manager|dashboard|prometheus|grafana|alertmanager|tf (infra|dns|ingress)")
+//flags passed to the command
+var flags *logFlags
+
+func newCmdLogs() *cobra.Command {
+	flags = newLogsFlags()
+	cmd := &cobra.Command{
+		Use:   "logs (gardener-apiserver|gardener-controller-manager|ui|api|scheduler|controller-manager|etcd-operator|etcd-main[etcd backup-restore]|etcd-main-backup|etcd-events[etcd backup-restore]|addon-manager|vpn-seed|vpn-shoot|machine-controller-manager|dashboard|prometheus|grafana|alertmanager|tf (infra|dns|ingress)",
+		Short: "Show and optionally follow logs of given component\n",
+		Long:  `s`,
```

Comment:
`Long` is the long description(https://godoc.org/github.com/spf13/cobra#Command) of the command, so I do not get why it is set to `s` ?
/cc @DockToFuture